source/features/rgh-feature-descriptions.tsx

```diff
@@ -94,7 +94,7 @@ async function init(): Promise<void | false> {
 
 	const conversationsUrl = '/sindresorhus/refined-github/issues?q=' + encodeURIComponent(`"${feature.id}" sort:updated-desc`);
 
-	const commitInfoBox = (await elementReady('.Box-header--blue.Details, include-fragment.commit-loader'))!.parentElement!;
+	const commitInfoBox = (await elementReady('.hx_commit-tease'))!.parentElement!;
```

Comment:
You should not replace the entire selector, as `.Box-header--blue.Details` is needed for older versions of GitHub Enterprise and `include-fragment.commit-loader` is needed according to [`2683e30` (#3852)](https://github.com/sindresorhus/refined-github/pull/3852/commits/2683e300976cacbbb71eb219347cb82aa964616b).

```suggestion
	const commitInfoBox = (await elementReady('.Box-header.Details, .Box-header--blue.Details, include-fragment.commit-loader'))!.parentElement!;
```

Reply:
`rgh-feature-descriptions` does not run on GHE 😃

Reply:
Ok, sure

Reply:
> `rgh-feature-descriptions` does not run on GHE smiley

Ohk ... let me know which ones to keep then

Reply:
> [`rgh-feature-descriptions`](/sindresorhus/refined-github/blob/main/source/features/rgh-feature-descriptions.tsx) does not run on GHE 😃

I forgot that 🤣

---

Quoting https://github.com/sindresorhus/refined-github/pull/4559#discussion_r667723258:

> - replace `.Box-header` with `.hx_commit-tease` (it seems to be a unique class name on the relevant pages)

Is there anything prevent us from using `.hx_commit-tease`?